### ISSUE TITLE
Yet another DoH resolver

### ISSUE THREAD

**Issue by bortzmeyer** [netblue30/fdns#18]
In your Server list, you may want to add https://doh.bortzmeyer.fr 

The no-logging policy here (in french)  :  https://doh.bortzmeyer.fr/policy

**Comment by startx2017** [comment]
Thank you! I put a link to your blog as a description - https://www.bortzmeyer.org/doh-mon-resolveur.html. If you want, we can change it.

**Comment by startx2017** [comment]
Sorry, replaced it with a better one: https://www.bortzmeyer.org/doh-bortzmeyer-fr-policy.html

**Comment by bortzmeyer** [comment]
Yes, the fist link was the technical description, most users will probably be more interested in the new one, describing the policy.

I suggest there is a bug in etc/servers, the URL for the service is https://doh.bortzmeyer.fr/, not https://doh.bortzmeyer.fr/dns-query (although a bug in dnsdist makes the second work as well but it's temporary). https://github.com/PowerDNS/pdns/issues/8573

**Comment by rusty-snake** [comment]
@bortzmeyer c515983

**Comment by netblue30** [comment]
Let's track this bug for now. We have quite a umber of dnsdist servers on the list.

https://github.com/PowerDNS/pdns/issues/8573
